frontend/src/scenes/notebooks/Notebook/notebookLogic.ts

```diff
@@ -417,7 +430,10 @@ export const notebookLogic = kea<notebookLogicType>([
                 return
             }
             const jsonContent = values.editor.getJSON()
+            const textContent = values.editor.getText()
+
             actions.setLocalContent(jsonContent)
+            actions.setLocalTextContent(textContent)
```

Comment:
Really don't love this... Having two states that are almost the same but not feels like bugs waiting to happen. Either we should derive them from the same actions or we should see if there is a different way to serialise the jsonContent (I bet there is)